bigml/logistic.py

```diff
@@ -283,6 +295,8 @@ def category_probability(self, input_data, unique_terms, category):
         for field_id in input_data :
             coefficients = self.get_coefficients(category, field_id)
             probability += coefficients[0] * input_data[field_id]
+            norm2 += math.pow(input_data[field_id], 2)
```

Comment:
so input_data contains only numbers here, right? i'm just a bit unsure because of the name, which in other places means just a the user input_data in a prediction request....